Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceListener.java

```diff
@@ -129,6 +136,8 @@ private void onIsolatePaused(@NotNull final IsolateRef isolateRef,
       final boolean suspend = myDebugProcess.getSession().breakpointReached(xBreakpoint, logExpression, suspendContext);
       if (!suspend) {
         myDebugProcess.getVmServiceWrapper().resumeIsolate(isolateRef.getId(), null);
+      } else {
+        myDebugProcess.getSession().positionReached(suspendContext);
```

Comment:
Hmm, I suppose I was unaware that breakpointReached() implied positionReached()? In any case, thanks for the  review! I'll re-test once this lands.